app/views/cms/collection.html.erb

```diff
@@ -0,0 +1,33 @@
+<%= render 'components/hero', hero_title: @title.titlecase, small_title: true %>
+<div class="govuk-width-container">
+  <%= render BreadcrumbComponent.new(ancestors: [[root_path, 'Home']], current_page_name: @page_name) %>
+  <div class="govuk-main-wrapper govuk-main-wrapper--xl">
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-full <%= @style_slug %>">
+      <div class="<%= @collection_wrapper_class %>">
+        <% @collection.resources.each do |resource| %>
+          <%= render resource.collection_view %>
+        <% end %>
+      </div>
+    </div>
+
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-full">
+        <% if @collection.previous_page %>
+          <hr class="ncce-hr">
+          <%= link_to "?page=#{@collection.previous_page}", class: 'icon-arrow-right search__paging' do %>
```

Comment:
Could you use more specific routes, ie: `foo_bar_path(page: @collection.previous_page})`